.circleci/config.yml

```diff
@@ -295,12 +295,11 @@ commands:
           command: |
             if [[ -n "$CIRCLE_TAG" ]]; then
               if [[ $CIRCLE_TAG == v* ]]; then
-                POM_VERSION_NAME=`echo $CIRCLE_TAG | sed s/v//`
+                sed -i -e "s/^VERSION_NAME=.*/VERSION_NAME=${CIRCLE_TAG:1}/" gradle.properties
```

Comment:
There's no `VERSION_NAME` in `gradle.properties` as things stand, so this `sed` doesn't do anything, does it?